test/tests/pbs_qstat_performance.py

```diff
@@ -121,11 +113,11 @@ def submit_and_stat_jobs(self, number_jobs):
         Submit specified number of simple jobs and stats jobs
         """
         self.submit_jobs(TEST_USER1, number_jobs)
-        for query in self.elapsed_time_dictionary:
+        for query in self.qstat_query_list:
             try:
                 self.compute_elapse_time(query)
             except RuntimeError as exception:
-                self.fail("Exception caught: " + str(exception))
+                self.assertfail("Exception caught: " + str(exception))
```

Comment:
I mean use assertTrue or assertFalse instead assertfail.... because assertfail is internal method to unittest.